ntoskrnl/io/iomgr/deviface.c

```diff
@@ -282,185 +282,239 @@ IopSeparateSymbolicLink(
 
 static
 NTSTATUS
-OpenRegistryHandlesFromSymbolicLink(IN PUNICODE_STRING SymbolicLinkName,
-                                    IN ACCESS_MASK DesiredAccess,
-                                    IN OPTIONAL PHANDLE GuidKey,
-                                    IN OPTIONAL PHANDLE DeviceKey,
-                                    IN OPTIONAL PHANDLE InstanceKey)
+IopOpenOrCreateSymbolicLinkSubKeys(
```

Comment:
It may be interesting to add a _minimal_ doxygen documentation block for this function and the OpenRegistryHandles..... one below.
(Maybe not for documenting all the parameters, there are a lot of them, but at least to explain the purpose of these functions.)

Reply:
Thanks. Added the short documentation for this and the 2nd function as well.